fastlane/lib/fastlane/actions/ssh.rb

```diff
@@ -6,7 +6,7 @@ module SharedValues
     end
 
     class SshAction < Action
-      def self.ssh_exec!(ssh, command)
+      def self.ssh_exec!(ssh, command, log)
```

Comment:
Fair enough. This is [fixed][1].

[1]: https://github.com/fastlane/fastlane/pull/4662/commits/f390e5dcca9028fe75dd6474835810bb84612780